src/common/components/CoreLayout/Sidebar/Sidebar.tsx

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import {
+  AfterclassIcon,
+  BookLineIcon,
+  DealsIcon,
+  StarLineAltIcon,
+} from "../../CustomIcon";
+import Link from "next/link";
+import { Icon } from "@iconify-icon/react";
+import { SidebarItem } from "./SidebarItem";
+import { useRouter } from "next/router";
+import { type SidebarListItem } from "./types";
+
+export const Sidebar = () => {
+  const router = useRouter();
+
+  return (
+    <div className="h-full w-60 space-y-6 border-r border-r-border-elevated px-4 py-5">
+      <Link href="/" className="flex items-center px-3 text-primary-default">
+        <AfterclassIcon />
```

Comment:
Should we replace this with the `<Logo>` component you've created? We can keep the text in there